Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -89,6 +89,7 @@ function vtkOpenGLImageMapper(publicAPI, model) {
         !oldOglRenderWindow.isDeleted() &&
         oldOglRenderWindow !== model._openGLRenderWindow
       ) {
+        // We unregister ourself when the render window changes
```

Comment:
Unregister the mapper when the render window changes.

(we tend to avoid using "we", "I", "ourself" in comments).